trace-normalization/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "datadog-trace-normalization"
+version = "0.0.0"
+authors = ["David Lee <david.lee@datadoghq.com>"]
+edition = "2021"
+
+[workspace]
```

Comment:
The workspace field shouldn't be necessary, the module if already inside a workspace, (defined in the cargo.toml as the root of the repo).